src/coreclr/tools/Common/TypeSystem/Ecma/EcmaField.cs

```diff
@@ -147,11 +146,6 @@ private int InitializeFieldFlags(int mask)
                 if ((fieldAttributes & FieldAttributes.HasFieldRVA) != 0)
                     flags |= FieldFlags.HasRva;
 
-#pragma warning disable SYSLIB0050 // Legacy serialization infrastructure is obsolete
-                if ((fieldAttributes & FieldAttributes.NotSerialized) != 0)
-                    flags |= FieldFlags.NotSerialized;
```

Comment:
Should we also remove the `FieldFlags.NotSerialized`? It should not be used now... and we don't want to use it again.

Reply:
Right - I'm... blind. Sorry.